yt/docs/README.md

```diff
@@ -1,208 +1,434 @@
+Individual sections can be included in or excluded from the document, depending on the values of variables. To describe visibility conditions, the `when` parameter is used. 
 
+> **Why the `when: audience == "public"` parameter is set?**</br>YTsaurus documentation is represented in two versions: public version (the one you are reading), and the internal one which is available only for the team members. As YTsaurus documentation uses the single source approach, both public and internal versions are built from the common content. So that, all articles for which the `when` parameter is not specified in `toc.yaml` appear in both internal and public documentation. If the `when: audience == "public"` parameter is specified for an article, it means that this article will only appear in the public documentation. Keep this in mind: when adding a new article, you should set the `when: audience == "public"` parameter.
 
-# Как написать новую страницу или внести изменения
+For more details about the table of contents, see the [YFM official documentation](https://diplodoc.com/docs/en/project/toc).
 
-1. Создаем страницу, например `test_page.md` в папке **arcadia/yt/docs**:
+------
 
-   ```bash
-   touch arcadia/yt/docs/test_page.md
-   ```
+### Variables
 
-2. Добавляем ее в SVN и выставляем тип  `text/markdown`:
+You may encounter the `{{}}` construction in the source, for example:
 
-   ```bash
-   svn add test_page.md
-   svn propset svn:mime-type 'text/markdown' test_page.md
-   ```
+```
+A {{product-name}} cluster is a group of instances located in containers ...
+```
 
-3. Наполняем страницу текстом. Предварительно рекомендуется прочесть раздел [Общие правила оформления документации](#Общие правила оформления документации), который находится ниже;
+The `{{product-name}}` is a variable that will be automatically substituted when rendering documentation. Variable values are specified in the `presets.yaml` files located in the `ytsaurus/yt/docs/ru` and `ytsaurus/yt/docs/en` directories. Below is the example:
 
-4. Добавляем файл в оглавление `toc.yml`;
+```
+# The presets.yaml file.
+default:
+    product-name: YTsaurus
+    audience: public
+    source-root: https://github.com/ytsaurus/ytsaurus/tree/main
+    ...
+```
 
-5. Собираем файл и проверяем его локально:
+You can read more about using the variables in the official [YFM official documentation](https://diplodoc.com/docs/en/project/presets).
 
-   ```bash
-   mkdir docs
-   ya make
-   tar -xzf docs.tar.gz -C docs
-   python3 -m http.server 8888 -d docs
-   или
-   cd docs
-   python ../dev-server.py
-   ```
+-------
 
-   Либо делаем pull request:
+### Reusing Content
```

Comment:
Reused content? Shared content?
This is not a section on how to reuse content, but rather on how our documentation is structured with a single shared source.